intezer_sdk/errors.py

```diff
@@ -77,6 +72,26 @@ def __init__(self, response: requests.Response):
         super().__init__('Index operation failed', response)
 
 
-class SubAnalysisOperationStillRunning(IntezerError):
+class OperationStillRunning(IntezerError):
```

Comment:
create an alias with the old name since this is breaking changes. Rename to `OperationStillRunningError`